src/msg/async/Stack.h

```diff
@@ -259,12 +273,35 @@ class Worker {
 
     perf_logger = plb.create_perf_counters();
     cct->get_perfcounters_collection()->add(perf_logger);
+
+    // Add labeled perfcounters
+    std::string labels = ceph::perf_counters::key_create(
+      name_prefix, {{"id", std::to_string(id)}});
```

Comment:
Be consistent with your indentation:
```suggestion
        name_prefix, {{"id", std::to_string(id)}});
```